src/windows/views/emojis_listview.py

```diff
@@ -109,48 +106,49 @@ def add_file(self, filepath):
             # Log exception
             log.warning("Failed to import file: {}".format(str(ex)))
 
+    @pyqtSlot(int)
+    def group_changed(self, index=-1):
+        log.info("Group set to: {}".format(index))
+        item = get_app().window.emojiFilterGroup.itemData(index)
+        log.info("Filtering on {}".format(item))
+        self.group_model.setFilterFixedString(item)
+        self.group_model.setFilterKeyColumn(1)
+
+        self.refresh_view()
+
     @pyqtSlot(str)
-    def filter_changed(self):
+    def filter_changed(self, filter_text=None):
+        """Filter emoji with proxy class"""
+        log.info("Filter text: {}".format(filter_text or ""))
+        self.model.setFilterRegExp(QRegExp(filter_text, Qt.CaseInsensitive))
+        self.model.setFilterKeyColumn(0)
```

Comment:
Making the `QRegExp` case insensitive automatically takes care of `setFilterCaseInsensitive()`. In my testing, the reverse was _not_ the case.

Reply:
Also not outdated.